scripts/clean-apps.ts

```diff
@@ -1,3 +1,4 @@
+#!node_modules/.bin/ts-node
```

Comment:
suggesting adding some shebangs to shorten a bit the yarn scripts in package.json

Reply:
and because shebang for `scripts` is a common practice: one should not need to know what binary the script should need to be called with

Reply:
Interesting! Thanks 🙏🏽